vin/app/models/subscriber.rb

```diff
@@ -1,44 +0,0 @@
-#enconding: utf-8
-
-class Subscriber < ActiveRecord::Base
-
-  # Attributes
-  # attr_accessible :name, :email, :phone, :address_id, :fb, :tw
-
-  # Associations
-  has_many :shipments
-  has_many :packages, through: :shipments
-  has_one :address
-
-  # Validations
-  validates_presence_of :name, :email, :phone, :address
-  validates_uniqueness_of :email
-  validates :name, length: { minimum: 1 }
-  validates :phone, length: { minimum: 4 }
-  validates :email, format: { with: /\A([^@\s]+)@((?:[-a-z0-9]+\.)+[a-z]{2,})\Z/i }, uniqueness: { case_sensitive: false }
-
-  # Scopes
-  scope :by_email, lambda { |email| where('email like ?', "%#{email}") }
-
-  # ClassMethods
-  def self.search(email)
-    if email.blank?
-      find(:all)
-    else
-      by_email
-    end
-  end
-
-  # Methods
-  def email_formater(email)
```

Comment:
I still don't know where to use it, i want all new emails to call `email_formatter` before getting in the database. Is just a trick to avoid duplicate subscriptors using gmail accounts with "+" or "."